# Check-in 1
Initially we tried to use LeafCutter as our main software, but due to compatibility issues it cannot be installed successfully on MacOS. We also attempted the installation on Windows but to no avail. Therefore, we switched to DEXSeq, an R package that focuses on exon counts. We next found a paper that performs differential analysis of splice junctions and intron retentions using DEXSeq on RNA-seq data from *Drosophila*. We aim to first familiarize ourselves with DEXSeq using processed data from the *pasilla* package, and then extend to the original dataset and try to replicate the tables and figure from the paper. 
## Addressing Feedback  
The published figure and tables are now displayed normally in README.md.

After looking into the documentation of DEXSeq and following its standard workflow, we decided to focus on the *Drosophila* GSE18508 dataset first, starting with differential exon usage and then maybe switching to splice junctions and intron retentions. We hope to identify significant genes and genome-wide patterns by mapping them to the genome. We could also perform gene ontology analyses to see if any clusters pop out. Through these processes, we wish we could be prepared for exploring the more complex GTEx dataset.  
## Progress
![](../fig/Fit_diagnostics.png)  
![](../fig/MA_plot.png)  
![](../fig/Fitted_FBgn0010909.png)  
Using the *pasilla* package, which processes a subset of the GSE18508 dataset with selected genes, we followed the standard analysis workflow of DEXSeq and generated the above figures (from top to bottom): fit diagnostics that provides an estimation of the variability within the data, MA plot that illustrates how the ability to detect differential exon usage depends on the number of reads mapped to an exon, and the differential exon usage profile of a specific gene, FBgn0010909, with isoforms marked.

We will then explore the data within the *pasilla* package further and try to extend the analysis workflow to the whole GSE18508 dataset. However, since *pasilla* has all the sequence alignment and annotation steps completed, we will need to adapt the pipeline of *pasilla* to GSE18508 and generate the ***DEXSeqDataSet***, the input of DEXSeq, ourselves, which could be challenging.  
## Project Organization
      ...  
      |-- README.md    <-- title, description, overview figure  
      |-- data/        <-- only small (<1 Mb), instructions on retrieving larger  
      |-- doc/         <-- reports, slidedecks  
      |-- results/     <-- only small (<1 Mb), instructions on retrieving larger  
      |-- src/         <-- code  
      ...
## Struggle
1. DEXSeq was designed to work with small sample numbers, so we are unsure if it would be feasible to process GTEx dataset with DEXSeq.  
2. Generating ***DEXSeqDataSet*** from GSE18508 requires alignment, annotation, and counting reads, which is challenging.
