# Check-in 2  
Since the last check-in we have worked on two directions: standardizing the workflow of converting raw RNA-seq data to ***DEXSeqDataSet***, and applying the example DEXSeq pipeline to analyse a subset of the GTEx dataset.
## Addressing Feedback  
We were able to circumvent the trouble of generating a ***DEXSeqDataSet*** ourselves by using the preloaded ***DEXSeqDataSet*** in the *pasilla* package, and FBgn0010909 is one of the genes in it.  

The red dots in the MA plot represents significant hits at an FDR=0.1. Captions are now updated in Check-in 1. The suggestion of choosing a subset of the GTEx dataset is very helping and is exactly what we are trying to do right now.  

For the FDR, we assume the resulting hits would increase or decrease in the same direction as the change in FDR threshold. The question concerning the choice of crucial genes is very relevant. Ideally, we are looking for genes that have high expression globally, but at the meantime also have a very tissue-specific isoform expression profile.  
## Progress  
![](../fig/published/MED12_EX.svg)  
The code for processing raw RNA-seq data is nearly completed, as well as the process of subsetting GTEx. Both are in the /src folder. 

For the GTEx part, we plan to start with 1000 genes across 20 samples and adjust the dimensions depending on the processing time. To minimize the effect of sampling error, we wish to randomize the choice of samples and genes, although some genes with isoforms of high tissue-specificity can be examined separately. MED12 is shown above as an example (plot generated from [GTEx Transcript Browser](https://gtexportal.org/home/transcriptPage))
## Project Organization  
Added fig folder for storing figures.  
## Struggle
1.  The exon count file from GTEx is of the parquet format and cumbersome to work with due to its bulky size.  
2. 
