# Check-in 2  
Since the last check-in we have worked on two directions: standardizing the workflow of converting raw RNA-seq data to ***DEXSeqDataSet***, and applying the example DEXSeq pipeline to analyse a subset of the GTEx dataset.
## Addressing Feedback  
We were able to circumvent the trouble of generating a ***DEXSeqDataSet*** ourselves by using the preloaded ***DEXSeqDataSet*** in the *pasilla* package, and FBgn0010909 is one of the genes in it.  

The red dots in the MA plot represents significant hits at an FDR=0.1. Captions are now updated in Check-in 1. The suggestion of choosing a subset of the GTEx dataset is very helping and is exactly what we are trying to do right now.  

For the FDR, we assume the resulting hits would increase or decrease in the same direction as the change in FDR threshold. The question concerning the choice of crucial genes is very relevant. Ideally, we are looking for genes that have high expression globally, but at the meantime also have a very tissue-specific isoform expression profile.  
## Progress  

## Project Organization  
Added fig folder for storing figures.  
## Struggle
1. DEXSeq was designed to work with small sample numbers, so we are unsure if it would be feasible to process GTEx dataset with DEXSeq.  
2. Generating ***DEXSeqDataSet*** from GSE18508 requires alignment, annotation, and counting reads, which is challenging.
